install/auto_installer.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+echo "======================================================"
+echo "Welcome to Mellivora CTF auto-installer"
+echo "Auto Installer works on any Ubuntu 16.04 server setup."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start the installation."
+echo ""
+
+# Updating & Upgrading
+
+sudo apt-get update && sudo apt-get -y upgrade
+
+echo "======================================================"
+echo "Select **LAMP server** with space and then follow prompts."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start"
+echo ""
+
+sudo apt-get -y install tasksel && sudo tasksel
+
+# Installing nano instead of vim
+
+sudo apt-get -y install nano
```

Comment:
Would be really great if you could choose between vim and nano

Reply:
Noted

Reply:
Use `$EDITOR` by default, if exists. It's preferred way, since it is end-user choice.